internal/rest/avi_obj_vsvip.go

```diff
@@ -606,13 +606,13 @@ func (rest *RestOperations) AviVsVipCacheAdd(rest_op *utils.RestOp, vsKey avicac
 		}
 
 		k := avicache.NamespaceName{Namespace: rest_op.Tenant, Name: name}
+		oldVsVipCache, oldVsVipFound := rest.cache.VSVIPCache.AviCacheGet(k)
```

Comment:
@aaha97: I did not get how status update will have impact by moving this line here?

Reply:
we are updating the cache on line 610, so calling old cache on line 615 will fetch updated cache and not old cache